example/.mage/tools.mk

```diff
@@ -1,49 +1,11 @@
 mage_folder := $(abspath $(dir $(lastword $(MAKEFILE_LIST))))
```

Comment:
minor: let's prefer `dir` over `folder` for consistency?

Reply:
Fixed